provider/datagen/src/driver.rs

```diff
@@ -598,47 +575,26 @@ impl DatagenDriver {
                     "Datagen configured without fallback with these locales: {:?}",
                     sorted_locale_strs
                 );
-                (false, DeduplicationStrategy::None)
+                DeduplicationStrategy::None
             }
             LocalesWithOrWithoutFallback::WithFallback { options, families } => {
-                let uses_internal_fallback = match options.runtime_fallback_location {
-                    None => sink.supports_built_in_fallback(),
-                    Some(RuntimeFallbackLocation::Internal) => true,
-                    Some(RuntimeFallbackLocation::External) => false,
-                };
-                let deduplication_strategy = match options.deduplication_strategy {
-                    // TODO(2.0): Default to RetainBaseLanguages here
```

Comment:
what does this TODO refer to then?

Reply:
It meant that BakedProvider should default to RetainBaseLanguages (by replacing where it currently says Maximal to instead say RetainBaseLanguages)